pages/common/git-push.md

```diff
@@ -17,3 +17,7 @@
 - Remove remote branches which don't exist locally
 
 `git push --prune {{REMOTE-NAME}}`
+
+- Publish tags
```

Comment:
It is called "publish" everywhere except places where `git push` removes remote branch.
Could you be more specific on what you think it should be?